feat(react-scheduler): add capability to edit recurring appointments (#2168)
DevExpress/devextreme-reactive: packages/dx-react-scheduler/docs/guides/editing.md

The following plugins implement this feature:

- [EditingState](../reference/editing-state.md) - controls the editing state
- [IntegratedEditing](../reference/integrated-editing.md) - implements editing

> - [EditingMenu](../reference/integrated-editing.md) - renders the editing menu and allows you to process user edits

Reviewer: Why 'EditingMenu' and not Edit Menu?

## Drag-and-Drop Editing

> Add the [DragDropProvider](../reference/drag-drop-provider.md) plugin to enable users to drag and drop appointments. Use its `allowDrag` and `allowResize` properties to disallow dragging and resizing specific appointments. You can also add the `EditingMenu` plugin to allow users to specify how they want to edit recurrent appointments.

Reviewer: ```suggestion
Add the [DragDropProvider](../reference/drag-drop-provider.md) plugin to enable users to drag and drop appointments. Use its `allowDrag` and `allowResize` properties to disallow dragging and resizing specific appointments. You can also add the `EditingMenu` plugin to allow users to specify how recurrent appointments are edited.
```
